relay-general/src/store/normalize/contexts.rs

```diff
@@ -177,7 +177,82 @@ fn normalize_os_context(os: &mut OsContext) {
     }
 }
 
+// TODO: extract urlencoded_from_str, parse_raw_response_data and normalize_response_data
+// can probably be reused in other places (original from normalize/request)
```

Comment:
Where would make sense to put this?
This is now being used by `Request` and `Response`.